docs/tutorials/01_data_structure.md

```diff
@@ -0,0 +1,154 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.0
+  kernelspec:
+    display_name: nocturne-research
+    language: python
+    name: python3
+---
+
+## Data format of a traffic scene
+
+This notebook dives into the data format used to create simulations in Nocturne.
+
+_Last update: April 2023_
+
+```python
+import json
+import matplotlib.pyplot as plt
+import seaborn as sns
+import pandas as pd
+
+cmap = ['r', 'g', 'b', 'y', 'c'] 
+%config InlineBackend.figure_format = 'svg'
+sns.set('notebook', font_scale=1.1, rc={'figure.figsize': (8, 3)})
+sns.set_style('ticks', rc={'figure.facecolor': 'none', 'axes.facecolor': 'none'})
+```
+
+Traffic scenes are constructed by utilizing the Waymo Open Motion dataset. Though every scene is unique, they all have the same basic data structure. 
+
+To load a traffic scene:
+
+```python
+# Take an example scene
+data_path = 'data/example_scenario.json'
+
+with open(data_path) as file:
+    traffic_scene = json.load(file)
+
+traffic_scene.keys()
+```
+
+### Global Overview 
+A traffic scene consists of:
+- `name`: the name of the traffic scenario.
+- `objects`: the road objects or moving vehicles in the scene.
+- `roads`: the road points in the scene, these are all the stationary objects.
+- `tl_states`: the states of the traffic lights, which are filtered out for now. 
+
+```python
+traffic_scene['tl_states']
+```
+
+```python
+traffic_scene['name']
+```
+
+```python
+pd.Series(
+    [
+        traffic_scene['objects'][idx]['type']
+        for idx in range(len(traffic_scene['objects']))
+    ]
+).value_counts().plot(kind='bar', rot=45, color=cmap);
+plt.title(f'Distribution of road objects in traffic scene. Total # objects: {len(traffic_scene["objects"])}');
+```
+
+This traffic scenario only contains vehicles and pedestrians, some scenes have cyclists as well.
+
+```python
+pd.Series(
+    [
+        traffic_scene['roads'][idx]['type']
+        for idx in range(len(traffic_scene['roads']))
+    ]
+).value_counts().plot(kind='bar', rot=45, color=cmap);
+plt.title(f'Distribution of road points in traffic scene. Total # points: {len(traffic_scene["roads"])}');
+```
+
+### In-Depth: Road Objects
+
+This is a list of different road objects in the traffic scene. For each road object, we have information about its position, velocity, size, in which direction its heading, whether its a valid object, the type, and the final position of the vehicle.
```

Comment:
nit: should be `in which direction it's heading, whether it's a valid object` as these are not possessives but contractions of `it is`.